### ISSUE TITLE
Process registration: `handle_connect` not being called?

### ISSUE THREAD

**Issue by stevegrossi** [hedwig-im/hedwig#59]
I've been following the README down through the [Registering your robot process](https://github.com/hedwig-im/hedwig#registering-your-robot-process) section in order to register a robot's process by name, but `handle_connect` doesn't appear to be called and the registration never happens.

This is with the latest master, on Elixir 1.3.4 and Erlang 19. A demo repo is here: https://github.com/stevegrossi/alfred. There's nothing special: to reproduce the issue I hewed as close to the README as possible, using `mix hedwig.gen.robot` with the name "alfred" and the `Console` adapter.

For simplicity, I'm attempting to start the robot manually, rather than through a supervision tree. Again, copy-and-pasting from the README's [Finding Your Robot](https://github.com/hedwig-im/hedwig#finding-your-robot) section:

<img width="761" alt="screen shot 2016-11-17 at 5 46 53 am" src="https://cloud.githubusercontent.com/assets/664341/20386597/529fb480-ac8a-11e6-877a-ade389d08e47.png">

The generated process registration via `:global` in `handle_connect` does not appear to take place. And indeed, if I replace the auto-generated `handle_connect` with something that should obviously fail:

```elixir
defmodule Alfred.Robot do
  use Hedwig.Robot, otp_app: :alfred

  def handle_connect(state) do
    raise "handle_connect called"
  end
```

The robot starts just fine, which leads me to believe `handle_connect` isn't being called at all. Is there possibly an issue here?

**Comment by scrogson** [comment]
@stevegrossi, yes, you are right. Looks like I failed to update the `Console` adapter. Sorry about that. 

I'll get a fix pushed shortly. Thank you for the report!

**Comment by scrogson** [comment]
@stevegrossi, thanks so much for the report. I really appreciate that. I've pushed up a fix in 112614f.

**Comment by stevegrossi** [comment]
Confirmed working! Thanks for addressing this so quickly.
